README.md

```diff
@@ -34,38 +39,55 @@ Allows you to bind an [Invocable Script](#invocable-scripts) to a hotkey.
 
 > ⚠️ Changes made in the `Startup scripts` to the `window.customJS` object might get overridden. To avoid that follow [State](#state) tips.
 
-## Usage/Example
+## How to use CustomJS
 
-CustomJS works by writing javascript classes. Each file can only contain one class.
+**IMPORTANT: CustomJS works by writing JavaScript classes. Each file can only contain one class.**
 
-````
-// in vault at scripts/coolString.js
+#### 1. Start by deciding where you want to place your scripts, say, in `scripts/`. Now, put your `.js` file there, following this template:
+
+```javascript
+// in vault, at scripts/coolString.js
 class CoolString {
-    coolify(s) {
-        return `😎 ${s} 😎`
-    }
+  coolify(s) {
+    return `😎 ${s} 😎`;
+  }
 }
+```
 
+Note: You can change the class name and the function name, but remember, **you need both a class and a function**! Otherwise it will not work.
+
+#### 2. In the CustomJS settings, you can set individual files as a script, or you can set a folder that contains several JS files. It is seperate—you don't need both for the same file. (Note: If you use an individual file, make sure to add the path as well.)

```

Comment:
I agree this clarification is a good idea. Can you edit the `Settings` section instead of adding it here? I think that makes more sense.